docs/reference/dapis/chains/chains.json

```diff
@@ -1,743 +1,562 @@
 {
-  "arbitrum-goerli-testnet": {
-    "name": "arbitrum-goerli-testnet",
-    "fullName": "Arbitrum Goerli Testnet",
-    "id": "421613",
+  "ethereum": {
+    "id": "1",
+    "alias": "ethereum",
+    "name": "Ethereum",
+    "nativeToken": "ETH",
+    "testnet": false,
+    "explorerUrl": "https://etherscan.io/",
     "contracts": {
-      "AccessControlRegistry": "0x12D82f38a038A71b0843BD3256CD1E0A1De74834",
-      "AirnodeRrpV0": "0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd",
       "Api3ServerV1": "0x3dEC619dc529363767dEe9E71d8dD1A5bc270D76",
-      "DapiServer": "0x2b4401E59780e44d3b1Fd2D41FCb3047c830F286",
+      "AccessControlRegistry": "0x12D82f38a038A71b0843BD3256CD1E0A1De74834",
       "OwnableCallForwarder": "0x81bc85f329cDB28936FbB239f734AE495121F9A6",
       "ProxyFactory": "0xd9cB76bcAE7219Fb3cB1936401804D7c9F921bCD"
-    },
-    "nativeToken": "ETH",
-    "blockTime": null,
-    "logoPath": "https://market.api3.org/images/chains/Arbitrum-Goerli-Testnet.webp",
-    "testnet": true,
-    "explorerUrl": "https://goerli.arbiscan.io"
+    }
   },
-  "arbitrum-nova": {
```

Comment:
arbitrum-nova isn't present in the updated version, but I guess that makes sense since it [currently has an empty chain list](https://docs.api3.org/reference/dapis/chains/) and the Market doesn't show it.

Reply:
Yah I'm glad it goes away with he changes to the Axios build script. I was thinking to add something to exclude it so this saves me the trouble.